chunky/src/java/se/llbit/chunky/main/CommandLineOptions.java

```diff
@@ -258,6 +258,10 @@ public CommandLineOptions(String[] args) {
       options.force = true;
     });
 
+    registerOption("-reload-chunks", new Range(0), arguments -> {
```

Comment:
We should probably add this option to the [USAGE string](https://github.com/chunky-dev/chunky/blob/master/chunky/src/java/se/llbit/chunky/main/CommandLineOptions.java#L66), so that people know it is there.